pymeasure/adapters/visa.py

```diff
@@ -57,6 +57,11 @@ def __init__(self, resource_name, visa_library='', preprocess_reply=None, **kwar
             resource_name = "GPIB0::%d::INSTR" % resource_name
         self.resource_name = resource_name
         self.manager = pyvisa.ResourceManager(visa_library)
+        # We can make that one non-static and use self.resource_name and self.manager,
+        # then it would get smaller.
+        # A better name would be "clean_arguments"
```

Comment:
@msmttchr what do you prefer? Mind if I make this a class method, maybe _private to indicate internal use only, or do you see this being useful outside the class somewhere?

Reply:
I think that, according to your approach, this method doesn't make sense anymore, even as internal method. I would remove it altogether and replace it with inline code where is needed.